### PR TITLE
fix: documentation refs and notes for Typescript SDK and more

### LINKED ISSUE #447: Looker SDK not working with angular
I  am trying to integrate Looker Embedded Client SDK with angular so installed @looker/SDK and configured setting as per guideline provided on official npm page for @looker/sdk to access APIs.

But when I tried to run `ng serve` and access a few methods and classes the angular application started failing. The look
Error:
```
 ERROR in node_modules/@looker/sdk-rtl/lib/oauthSession.d.ts:1:23 - error TS2688: Cannot find type definition file for 'request'.

    1 /// <reference types="request" />
    
```

Configuration:
```
// package.json 

{ 
   //package.json
   
 "dependencies": {
    "@angular/core": "~10.2.3",
    "@looker/sdk": "^7.20.3",
    // few other libs
  },
  "devDependencies": {
   // few other libs  
  "@angular/cli": "~10.2.0",
     "typescript": "~4.0.5"
  },
 
//tsconfig.json
"compilerOptions": {
  // few other settings
  "module": "es2020",
  "target": "es5",
   "lib": [
    "es2020",
    "dom"
  ],
 

/ /And in component file:

import { Looker31SDK, LookerBrowserSDK } from '@looker/sdk/lib/browser'

@Component({
  selector: 'reports',
  templateUrl: './reports.component.html'
})
export class ReportsComponent implements OnInit {

  constructor() {
  }
  ngOnInit() {
    this.getLookerConfig()
      .then(response => {
        console.log(response, 'Success')
      }).catch(error => {
      console.log(error, 'Error')
    });
  }

  async getLookerConfig() {
    const sdk = LookerBrowserSDK.init31();
    // retrieve your user account to verify correct credentials
    const me = await sdk.ok(sdk.me(
      "id, first_name, last_name, display_name, email, personal_space_id, home_space_id, group_ids, role_ids"))
    console.log({me})
    // make any other calls to the Looker SDK
    const dashboards = await sdk.ok(
      sdk.search_dashboards({title: 'My SDK dashboard'})
    )
    if (dashboards.length === 0) {
      console.log('Dashboard not found')
    }


    await sdk.authSession.logout()
    if (!sdk.authSession.isAuthenticated()) {
      console.log('Logout successful')
    }
  }
```

### PR DESCRIPTION
**Pull request by jkaster**
Also address some reported issues

Closes #447, #487

### CODE REVIEW

File: packages/sdk/README.md [hunk 1]
```diff
@@ -38,13 +52,13 @@ All requests to the Looker API server require an access token. For browser imple
 
 Authenticating for the browser takes more setup than authenticating for use with a Node application.
 
-The stand-alone version of the [Looker API Explorer](/packages/api-explorer) uses OAuth and the `BrowserSDK` to get an authentication token for Looker API requests.
+The stand-alone version of the [Looker API Explorer](../../packages/api-explorer) uses OAuth and the `BrowserSDK` to get an authentication token for Looker API requests.
```
Comment (bryans99) [inline packages/sdk/README.md:55]: ../../packages/api-explorer can be ../api-explorer?

File: packages/sdk/README.md [hunk 2]
```diff
@@ -38,13 +52,13 @@ All requests to the Looker API server require an access token. For browser imple
...
 
-- [RunItSDK](/packages/run-it/src/utils/RunItSDK.ts) shows how to override `readConfig()` to get SDK configuration values.
+- [RunItSDK](../../packages/run-it/src/utils/RunItSDK.ts) shows how to override `readConfig()` to get SDK configuration values.
```
Comment (bryans99) [inline packages/sdk/README.md:57]: more possible cleanup of relative directories

**Review by bryans99 — COMMENTED** [review]
Minor observations. Can approved if needed

File: packages/api-explorer/CHANGELOG.md [hunk 3]
```diff
@@ -332,4 +332,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Response types now have nested types correctly documented
 - The response type processing functions now have arguments for the number of levels to expand nested types
 - API explorer's default expansion depth is one level deep
-- Added [`ExploreType`](/packages/api-explorer/src/components/ExploreType/ExploreType.tsx) and [`ExploreProperty`](/packages/api-explorer/src/components/ExploreType/ExploreProperty.tsx) components used to display responses and explore the type on the type page
\ No newline at end of file
+- Added [`ExploreType`](../../packages/api-explorer/src/components/ExploreType/ExploreType.tsx) and [`ExploreProperty`](../../packages/api-explorer/src/components/ExploreType/ExploreProperty.tsx) components used to display responses and explore the type on the type page
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/api-explorer/CHANGELOG.md:335]: good idea. That was a global search and replace thing.

File: examples/ruby/create_themes.rb [hunk 4]
```diff
@@ -7,9 +7,9 @@
   :api_endpoint => ENV['LOOKERSDK_BASE_URL']
 )
 ​
-# NOTE: Custom themes needs to be enabled by Looker. 
-# Unless custom themes are enabled, only the automatically generated default theme can be used. 
-# Please contact your Account Manager or support@looker.com to update your license for this feature.
+# NOTE: Custom themes needs to be enabled by Looker.
+# Unless custom themes are enabled, only the automatically generated default theme can be used.
+# Please contact your Account Manager or https://help.looker.com to update your license for this feature.
```
[the remark this answers is not in the document]
Comment (jkaster) [reply examples/ruby/create_themes.rb:12]: I didn't want to provide a localized link and `help.looker.com` is our goto replacement for `support@looker.com`

File: packages/sdk/README.md (hunk 1 above)
Comment (jkaster) [reply packages/sdk/README.md:55]: I should have had you do these updates 👍🏼

File: packages/sdk-node/README.md [hunk 5]
```diff
@@ -32,6 +32,20 @@ npm install @looker/sdk @looker/sdk-rtl @looker/sdk-node
 
 **Note**: If you are only intending to use the SDK in a browser, omit `@looker/sdk-node`.
 
+Some other dependencies may be required for your project to build and run correctly.
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-node/README.md:35]: do you mean the typescript examples readme here?

File: examples/ruby/create_themes.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (bryans99) [reply examples/ruby/create_themes.rb:12]: Perhaps something like: 
```
To update your license for this feature, please contact your Account Manager or visit https://help.looker.com for additional resources.
```
When I go to help.looker.com, I don't see anything related to licenses

File: packages/sdk-node/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (bryans99) [reply packages/sdk-node/README.md:35]: It looks okay now. Either I was imagining it or you fixed it

**Review by bryans99 — APPROVED** [review]
LGTM

**Pull request merged**